src/style.css

```diff
@@ -1,3 +1,133 @@
-body {
+* {
   margin: 0;
 }
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.image {
+  margin-left: 3px;
```

Comment:
Try to center your image without hardcode margin values, try to use some flexible values, because, for example, for a different product, the image may be of a different size and you will need to use not margin-left: 3px but margin-left: 6px